frontend/package.json

```diff
@@ -19,16 +20,28 @@
     "@angular/platform-browser": "~13.2.0",
     "@angular/platform-browser-dynamic": "~13.2.0",
     "@angular/router": "~13.2.0",
+    "angular2-uuid": "^1.1.1",
```

Comment:
![image](https://user-images.githubusercontent.com/75627690/164265507-3deb6523-33a6-4307-a0d4-285ae7b809b6.png)
![image](https://user-images.githubusercontent.com/75627690/164265554-e1ccd577-0421-4ed7-b59b-cab3144a71e0.png)

Why you don't use https://www.npmjs.com/package/uuid this library?

Reply:
done

Reply:
Weird. I still see the package.